scripts/dependencies/lima.ts

```diff
@@ -28,7 +28,7 @@ import { simpleSpawn } from 'scripts/simple_process';
  * @param version2 The second version to compare.
  * @returns Whether version1 is higher (-1), equal to (0), or lower than (1) version2.
  */
-function rcompareVersions(version1: string, version2: string): -1 | 0 | 1 {
```

Comment:
Might be worthwhile to move this function into somewhere more general; putting it under `lima` is confusing now.

Reply:
Into `scripts/lib/dependencies.ts`?

Reply:
Yeah, that sounds reasonable.